demos/twa-web-share-target/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this be below the copyright text?

Reply:
According to the XML spec, the XML declaration must be the first line of the document, including comments. In practice, having the comments before that does indeed break builds.